### PR TITLE
Added an UIImage extension to use FontAwesome as image

### PR DESCRIPTION
**Pull request by planetexpress69**
Hi Doan Truong Thi,

I recently came across your code and wanted to make an UITabBarItem's image from FontAwesome's icons. Would like to see this in your master branch.

Regards from Germany,
Martin

### CODE REVIEW

**Comment by krishaamer** [comment]
That's just what I needed, thank you!

**Comment by thii** [comment]
@planetexpress69 Thank you! I can buy you a 🍻 if you happen to be in the Dubdub this year.

**Comment by planetexpress69** [comment]
Welcome. Where’s Dubdub? :-)

File: Classes/FontAwesome.swift [hunk 1]
```diff
@@ -659,3 +659,23 @@ public extension String {
     return name.rawValue.substringToIndex(advance(name.rawValue.startIndex, 1))
   }
 }
+
+public extension UIImage {
+    public static func fontAwesomeIconWithName(name: FontAwesome, textColor: UIColor, size: CGSize) -> UIImage {
+        let paragraph = NSMutableParagraphStyle()
+        paragraph.lineBreakMode = NSLineBreakMode.ByWordWrapping
+        paragraph.alignment = .Center
+        let attributedString = NSAttributedString(string: String.fontAwesomeIconWithName(name) as String, attributes: [NSFontAttributeName: UIFont.fontAwesomeOfSize(24.0), NSForegroundColorAttributeName: textColor, NSParagraphStyleAttributeName:paragraph])
```
Comment (UglyTroLL) [inline Classes/FontAwesome.swift]: Why hardcoded the font size to 24?

Comment (planetexpress69) [inline Classes/FontAwesome.swift]: Reasonable size for usage in UITabBarItem, feel free to add another parameter in fontAwesomeIconWithName(…).

**Pull request merged**